Mage.Sets/src/mage/cards/u/UnfinishedBusiness.java

```diff
@@ -0,0 +1,123 @@
+package mage.cards.u;
+
+import java.util.UUID;
+
+
+import mage.abilities.Ability;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.*;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
+import mage.filter.StaticFilters;
+import mage.filter.predicate.Predicates;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetCardInYourGraveyard;
+
+/**
+ *
+ * @author Codermann63
+ */
+public final class UnfinishedBusiness extends CardImpl {
+
+    private static final FilterCard auraOrEquipmentFilter = new FilterCard("Aura or Equipment card");
+
+    static {
+        auraOrEquipmentFilter.add(Predicates.or(
+                SubType.EQUIPMENT.getPredicate(),
+                SubType.AURA.getPredicate()
+        ));
+    }
+
+    public UnfinishedBusiness(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{3}{W}{W}");
+        this.getSpellAbility().addTarget(new TargetCardInYourGraveyard(1,1, StaticFilters.FILTER_CARD_CREATURE));
+        this.getSpellAbility().addTarget(new TargetCardInYourGraveyard(0, 2, auraOrEquipmentFilter));
+        // Return target creature card from your graveyard to the battlefield,
+        // then return up to two target Aura and/or Equipment cards from your graveyard to the battlefield attached to that creature.
+        this.getSpellAbility().addEffect(new UnfinishedBusinessEffect());
+    }
+
+    private UnfinishedBusiness(final UnfinishedBusiness card) {
+        super(card);
+    }
+
+    @Override
+    public UnfinishedBusiness copy() {
+        return new UnfinishedBusiness(this);
+    }
+}
+
+class UnfinishedBusinessEffect extends OneShotEffect{
+
+    public UnfinishedBusinessEffect() {
```

Comment:
card-specific classes like this should be package-private (i.e. remove the `public` access modifier)